model_utils.py

```diff
@@ -72,6 +72,8 @@ def run(self):
         start = time.time()
         self.trained_model = None
 
+        print(f"\n xxx self.external_trained_model = {self.external_trained_model} / {self.trained_model}\n")
```

Comment:
self.trained_model should be the object resulting from loading the path stored in self.external_trained_model. are you just trying to check if self.trained_model is none here?

Reply:
huh? I only modified create_heatmaps_jobs.py ... I don't recall modifying anything else.

Reply:
oh shoot, I forgot that I had added a few debug statements in model_utils.py. Only accept change in create_heatmaps_job.py